src/app.js

```diff
@@ -1,69 +1,83 @@
 /**
  * Created by championswimmer on 28/5/15.
  */
-'use strict';
+(function () {
+"use strict";
 // Declare app level module which depends on views, and components
 
-var openevent = angular.module('openevent',
-    [
-        'ui.router',
-        'ngStorage',
-        'ngMaterial',
-        'leaflet-directive',
-        'oe.sidenav',
-        'oe.sessions',
-        'oe.speakers',
-        'oe.tracks',
-        'oe.sponsors',
-        'oe.twitterwall',
-        'oe.map'
-    ]);
-
-openevent.config(['$urlRouterProvider', '$httpProvider', '$mdThemingProvider',
-        function($urlRouterProvider, $httpProvider, $mdThemingProvider) {
-
-            //$httpProvider.defaults.useXDomain = true;
-            //delete $httpProvider.defaults.headers.common['X-Requested-With'];
-            $urlRouterProvider.otherwise('sessions');
-
-            $mdThemingProvider.theme('default')
-                .primaryPalette(config.themeColorPrimary)
-                .accentPalette(config.themeColorAccent);
+const openevent = angular.module("openevent",
+  [
+    "ui.router",
+    "ngStorage",
+    "ngMaterial",
+    "leaflet-directive",
+    "oe.sidenav",
+    "oe.sessions",
+    "oe.speakers",
+    "oe.tracks",
+    "oe.sponsors",
+    "oe.twitterwall",
+    "oe.map"
+  ]);
+
+openevent.config(["$urlRouterProvider", "$httpProvider", "$mdThemingProvider", 
+        function setTheme($urlRouterProvider, $httpProvider, $mdThemingProvider) {
+            //$httpProvider.defaults.useXDomain = true;   
+            //delete $httpProvider.defaults.headers.common["X-Requested-With"];
+          $urlRouterProvider.otherwise("sessions");
+          $mdThemingProvider.theme("default")
+            .primaryPalette(config.themeColorPrimary)
+            .accentPalette(config.themeColorAccent);
+
         }]);
 
-openevent.controller('AppController',
-    [ '$mdSidenav', '$mdMedia', '$sessionStorage', 'ApiJsonFactory',
+openevent.controller("AppController",
+    ["$mdSidenav", "$mdMedia", "$sessionStorage", "ApiJsonFactory",
         function($mdSidenav, $mdMedia, $sessionStorage, ApiJsonFactory) {
-    var app = this;
-    openevent.totalDays = 0;
-    app.appTitle = config.title;
-    app.toggleSidenav = function(menuId) {
-        $mdSidenav(menuId).toggle();
-    };
-
-    app.$storage = $sessionStorage;
-    if ( app.$storage.event === null ||
-        typeof(app.$storage.event) == 'undefined')
-    {
-        app.$storage.event = [];
-        app.$storage.days = [];
-    }
-    app.Event = app.$storage.event;
-
-    if (app.Event.length === 0) {
-        ApiJsonFactory.getJson('event')
+
+          const app = this;
+          
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2415208638/issues/source?bid=3233857&fileBranchId=3235518#l39)